README.md

```diff
@@ -1,23 +1,44 @@
+<!--
```

Comment:
I think you need two READMEs on top of this, one for each similar to how we have it in `module-data-storage` (https://github.com/gruntwork-io/module-data-storage/tree/master/modules/rds), so that we can have two service entries that map to this repo.

It does mean a ton of duplicated content though...

Reply:
Ok, will do that. @eak12913, if I create `README-MySQL.md` and `README-PostgreSQL.md`, do both of those need to have the metadata section on top?